gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/RemoteGcsHelper.java

```diff
@@ -89,6 +89,23 @@ public static Boolean forceDelete(Storage storage, String bucket, long timeout,
   }
 
   /**
+   * Deletes a bucket, even if non-empty. Objects in the bucket are listed and deleted until bucket
+   * deletion succeeds. This method can be used to delete buckets from within App Engine.  Note that
```

Comment:
Instead, I would mentioned in the other method that it uses a separate thread to allow the timeout
and if that is a problem (such as in AE) one should use this method instead (with a "@link" reference). Also, instead of "Note that ..." I would just say "This method blocks until the deletion competed or failed".